README.md

```diff
@@ -19,74 +18,88 @@
 2. Add the package to your `nuxt.config.ts`:
     ```ts
     export default defineNuxtConfig({
-      modules: ['@sidebase/nuxt-auth'],
-    })
+        modules: ["@sidebase/nuxt-auth"],
```

Comment:
why did you decide to reformat the whole readme? Please revert this

Reply:
yeah this is auto enabled at my end..
reverting this.